keps/sig-scalability/0001-enhance-network-performance-benchmark-testcases.md

```diff
@@ -0,0 +1,196 @@
+---
```

Comment:
Please adjust to the new KEP template (kep.yaml and README.md).

Reply:
@wojtek-t @aojea : I have updated the new KEP template. I have taken care most of the review comments. Please review.